### PR TITLE
Create the collection listing page layout

### PR DESCRIPTION
**Pull request by aerim-yi**
[Jira Ticket](https://immutable.atlassian.net/browse/II-58?atlOrigin=eyJpIjoiM2U3ZDkwZmY2ODgzNDYyNjgxZjNkODk5MjhjZDY0ZDMiLCJwIjoiaiJ9)

### CODE REVIEW

File: src/components/HeaderBar/HeaderBar.test.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { render} from '@testing-library/react'
+import HeaderBar from './HeaderBar'
+
+describe('NavBar component', () => {
+    test('match snapshot', () => {
+        const { container } = render(<HeaderBar />);
+        expect(container).toMatchSnapshot()
+    })
+})
```
Comment (maksim-tsvetkov) [inline src/components/HeaderBar/HeaderBar.test.tsx:9]: Add an empty line at the end of file.

File: src/components/HeaderBar/HeaderBar.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    return (
+        <Navbar bg="dark" variant="dark">
+            <Container>
+                <Navbar.Brand href="#home">Intern X Marketplace</Navbar.Brand>
+            </Container>
+        </Navbar>
+    )
+}
+
+export default HeaderBar
```
Comment (maksim-tsvetkov) [inline src/components/HeaderBar/HeaderBar.tsx:13]: Same here 🙈

File: src/pages/CollectionsPage.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+                    <Col>
+                    {/* This is where the cards will be displayed */}
+                    </Col>
+                </Row>
+            </Container>
+        </Fragment>
+    )
+}
+
+export default CollectionsPage
```
Comment (maksim-tsvetkov) [inline src/pages/CollectionsPage.tsx:20]: And here 🙈

**Review by maksim-tsvetkov — CHANGES_REQUESTED** [review]
Left a few tiny comments on the PR. 
Tip:
I don't want to slow down your work anyhow but if you have spare time it would be nice to read [the commit convention](https://www.conventionalcommits.org/en/v1.0.0/) and stick to the commit naming practices from the beginning. 
The following of this document would provide better visibility for reviewers and yourself on what has been included in the PR and the development steps, 
Having a clear development history is a significant aspect that helps to maintain the code base efficiently.

**Review by maksim-tsvetkov — APPROVED** [review]

**Comment by aerim-yi** [comment]
> 



> Left a few tiny comments on the PR. Tip: I don't want to slow down your work anyhow but if you have spare time it would be nice to read [the commit convention](https://www.conventionalcommits.org/en/v1.0.0/) and stick to the commit naming practices from the beginning. The following of this document would provide better visibility for reviewers and yourself on what has been included in the PR and the development steps, Having a clear development history is a significant aspect that helps to maintain the code base efficiently.

Thank you for this information. I'll improve my commit messages next time.

**Review by jinyoungkim927 — APPROVED** [review]

**Pull request merged**